FirebaseCore/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v6.7.0 -- M70
+- [fixed] Updated the nanopb version dependency across Firebase to 0.3.9.5 that
```

Comment:
You can phrase this more simply and tersely, while giving more details.

> Updated nanopb to 0.3.9.5 (across all Firebase pods). This includes a fix for [CVE-2020-5235](https://github.com/nanopb/nanopb/security/advisories/GHSA-gcx3-7m76-287p). Note that the versioning scheme for the nanopb CocoaPod has changed, see https://github.com/google/nanopb-podspec for more details.

In particular:
  * We should link to details of the specific vulnerability we're fixing
  * I don't think we need to fully spell out the versioning change here, given that we're linking to a page that describes it in detail